com.unity.netcode.gameobjects/Runtime/Messaging/Messages/OrderingMessage.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+
+namespace Unity.Netcode
+{
+    /// <summary>
+    /// This particular struct is a little weird because it doesn't actually contain the data
+    /// it's serializing. Instead, it contains references to the data it needs to do the
+    /// serialization. This is due to the generally amorphous nature of network variable
+    /// deltas, since they're all driven by custom virtual method overloads.
+    /// </summary>
+    internal struct OrderingMessage : INetworkMessage
```

Comment:
tests? 👀

Reply:
Added unit tests. For integration tests, we'd need too much overhaul.